index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='https://fonts.googleapis.com/css?family=Playfair+Display' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <title>Felix Odete's Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+    <header class="head-container">
+        <a href="#" class="home-logo">
+            <div class="logo">
+                <img src="img/yellow-triangle.svg" alt="" id="yellow-triangle">
+                <img src="img/red-square.svg" alt="" id="red-square">
+            </div>
+        </a>
+        <nav class="nav flex-box">
+            <a href="#">
+                <div class="menu-button">
+                    <div class="rectangle"></div>
+                    <div class="rectangle"></div>
+                    <div class="rectangle"></div>
+                </div>
+            </a>
+        </nav>
+    </header>
+    <div class="container">
+        <main>
+            <section>
+                <div class="name">
+                    <h1>Felix Odete</h1>
+                </div>
+                <div class="about-me">
+                    <p>Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                        If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+                </div>
+                <div class="rectangle-button">
+                    <h1>Start collaboration</h1>
```

Comment:
- [ ] [optional] Please replace this `<h1>` heading with other headings example H2, H3, etc. A webpage should only have one `<h1>` that describes what a webpage is about.